williamrijksen/com.williamrijksen.onesignal
# Fix/depreciation ids available

## Context
Replaces the idsavailable function by the get getPermissionSubscriptionState. 
 - [getPermissionSubscriptionState iOS](https://documentation.onesignal.com/docs/ios-native-sdk#section--getpermissionsubscriptionstate-)
 - [getPermissionSubscriptionState Android](https://documentation.onesignal.com/docs/android-native-sdk#section--getpermissionsubscriptionstate-)

## How to test
 - build the module
 - add the following code to your test device/application
```js
        var onesignal = require('com.williamrijksen.onesignal');
        
        var state = onesignal.getPermissionSubscriptionState();
alert(JSON.stringify(state));
```
- Verify the outcome is as expected.

## Notes
 Blocked by #62 
This is a breaking change!
